### DISCUSSION TITLE
Problems with Google Analytics

### DISCUSSION THREAD

**Discussion by simone-mariani** [HEIGE-PCloud/DoIt#677] [Q&A]
Hello. 

I am having some problems with the configuration of Google Analytics. 

I have this on my config.toml

```
#Google analytics
[params.analytics]
    enable = true
    # Google Analytics
    [params.analytics.google]
      id = "my id"
      anonymizeIP = true
```

But it is not working. 

Any ideas?

**Comment by CXwudi** [comment]
Hi, I figured that the ID must be the measurement ID, not stream ID. Maybe that is the case for you?
![image](https://user-images.githubusercontent.com/17377423/187014736-74e5207a-4c53-49ab-a18f-d98fbd864298.png)
